packages/run-protocol/src/getRUN.js

```diff
@@ -0,0 +1,338 @@
+    const maxAvailable = floorMultiplyBy(amountLiened, collateralPrice);
+    const collateralizedRun = ceilMultiplyBy(runWanted, collateralizationRatio);
+    assert(
+      AmountMath.isGTE(maxAvailable, collateralizedRun),
+      X`${amountLiened} at price ${collateralPrice} not enough to borrow ${runWanted} with ${collateralizationRatio}`,
+    );
+
+    return { runWanted, attestationGiven, amountLiened };
+  };
+
+  return harden({
+    getCurrentTerms: () => ({
+      collateralPrice: getRatio(CreditTerms.CollateralPrice),
+      collateralizationRatio: getRatio(CreditTerms.CollateralizationRatio),
+    }),
+    checkBorrow,
+    /** @param { ZCFSeat } seat */
+    checkOpenProposal: seat => {
+      assertProposalShape(seat, {
+        give: { Attestation: null },
+        want: { RUN: null },
+      });
+      const {
+        give: { Attestation: attAmt },
+        want: { RUN: runWanted },
+      } = seat.getProposal();
+
+      return checkBorrow(attAmt, runWanted);
+    },
+  });
+};
+
+/**
+ * @param { ContractFacet } zcf
+ * @param {{
+ *   feeMintAccess: FeeMintAccess,
+ *   initialPoserInvitation: Invitation,
+ *   lienBridge: ERef<StakingAuthority>,
+ * }} privateArgs
+ */
+const start = async (
+  zcf,
+  { feeMintAccess, initialPoserInvitation, lienBridge },
+) => {
+  const {
+    main: initialValue,
+    brands: { Stake: stakeBrand },
+    lienAttestationName = 'BldLienAtt',
+  } = zcf.getTerms();
+
+  const att = await makeAttestationFacets(
+    zcf,
+    stakeBrand,
+    lienAttestationName,
+    lienBridge,
+  );
+  const attestBrand = await E(att.publicFacet).getBrand();
+
+  const builder = makeParamManagerBuilder(zcf.getZoeService())
+    .addBrandedRatio(
+      CreditTerms.CollateralPrice,
+      initialValue[CreditTerms.CollateralPrice].value,
+    )
+    .addBrandedRatio(
+      CreditTerms.CollateralizationRatio,
+      initialValue[CreditTerms.CollateralizationRatio].value,
+    );
+  await builder.addInvitation(CONTRACT_ELECTORATE, initialPoserInvitation);
+  const paramManager = builder.build();
+  const { wrapPublicFacet, wrapCreatorFacet, getRatio } = handleParamGovernance(
+    zcf,
+    paramManager,
+  );
+
+  const runMint = await zcf.registerFeeMint('RUN', feeMintAccess);
+  const { brand: runBrand, issuer: runIssuer } = runMint.getIssuerRecord();
+  const creditPolicy = makeCreditPolicy(
+    { Attestation: attestBrand, RUN: runBrand, Stake: stakeBrand },
+    getRatio,
+  );
+
+  const revealRunBrandToTest = () => {
+    return harden({ runMint, runBrand, runIssuer });
+  };
+  zcf.setTestJig(revealRunBrandToTest);
+
+  /** @type { OfferHandler } */
+  const makeLineOfCreditHook = seat => {
+    return makeLineOfCreditKit(zcf, seat, creditPolicy, runMint, runBrand);
+  };
+
+  const publicFacet = wrapPublicFacet(
+    Far('getRUN Public API', {
+      getIssuer: att.publicFacet.getIssuer,
+      getBrand: () => att.publicFacet.getBrand,
```

Comment:
Doesn't this want to set getBrand to the getBrand function, rather than setting it to be a function that returns a function?  Or am I mis-reading it?
```suggestion
      getBrand: att.publicFacet.getBrand,
```

Reply:
that looks funny to me too. I suspect it's not tested.

Reply:
This is subsumed by using `getTerms()` to get brands and issuers.